frontend/src/features/projects/dispose/components/ExemptionRequest.tsx

```diff
@@ -0,0 +1,66 @@
+import './ExemptionRequest.scss';
+
+import * as React from 'react';
+import { Form } from 'react-bootstrap';
+import { Check } from 'components/common/form';
+import { TextArea } from 'components/common/form';
+import TooltipIcon from 'components/common/TooltipIcon';
+import { useFormikContext, getIn } from 'formik';
+
+export interface IProjectExemptionProps {
+  /** the label of the ExemptionRequest checkbox */
+  exemptionLabel?: string;
+  /** the label for the rationale notes */
+  rationaleInstruction?: string;
+  /** the tooltip to be included with the label */
+  tooltip?: string;
+  /** the field the exemption boolean is applied to */
+  exemptionField: string;
+  /** the rationale field the text field will be applied to */
+  rationaleField: string;
+  /** Header used at the start of the component */
+  sectionHeader?: string;
+}
+
+/**
+ * Component used for requesting an ERP exemption while submitting a project.
+ * @param param0
+ */
+export default function ProjectNotes({
```

Comment:
/nit: Could you please update the component default export to match the filename - i.e. `export default function ExemptionRequest(...)` ?

Reply:
oops